proto/ibc/applications/ibc_query/v1/query.proto

```diff
@@ -42,4 +42,5 @@ message QueryCrossChainQueryResultResponse {
   string id = 1;
   QueryResult result = 2;
   bytes data = 3;
+  string sender = 4;
```

Comment:
Why should the query requester receive sender field?

Reply:
I deleted this field, but I didn't add it to the commit.,Thank you for check